java/example_code/cloudmap/src/main/java/aws/example/cloudmap/CreateService.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Author : Rajib Deb
```

Comment:
You do not need this top comment block with your name and company identification. 

Your github username is associated with this code example in the sourceauthor tag

Reply:
This is done